### PR TITLE
round 2 of documentation fixes

### PR DESCRIPTION
**Pull request by haircommander**
This PR addresses some left over comments from the first round of documentation fixes

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM

**Comment by saschagrunert** [comment]
LGTM

File: tutorials/setup.md [hunk 1]
```diff
@@ -109,11 +106,16 @@ To add build tags to the make option the `BUILDTAGS` variable must be set.
...
-| apparmor  | apparmor profile support           | <none>      |
+| Build Tag                        | Feature                                         | Dependency   |
+|----------------------------------|-------------------------------------------------|--------------|
+| seccomp                          | syscall filtering                               | libseccomp   |
+| selinux                          | selinux process and mount labeling              | libselinux   |
+| apparmor                         | apparmor profile support                        | <none>       |
+| exclude_graphdriver_btrfs        | exclude btrfs as a storage option               | <none>       |
+| btrfs_noversion                  | for building btrfs version < 3.16.1             | btrfs        |
+| exclude_graphdriver_devicemapper | exclude devicemapper as a storage option        | <none>       |
+| libdm_no_deferred_remove         | don't compile deferred remove with devicemapper | devicemapper |
```
Comment (saschagrunert) [inline tutorials/setup.md]: We could also add the build tags for containers/image or refer to them: https://github.com/containers/image#supported-build-tags

Comment (haircommander) [reply tutorials/setup.md]: I sorted those and the ones for c/storage into their own table!

**Pull request merged**